frontend/src/pages/details-page.tsx

```diff
@@ -17,12 +17,12 @@ export default function DetailsPage() {
     const getPostById = async () => {
       try {
         await axios.get(import.meta.env.VITE_API_PATH + `/api/posts/${postId}`).then((response) => {
-          console.log(response.data);
+          // console.log(response.data);
           setIsLoading(false);
```

Comment:
Why uncomment?  we would be needing that

Reply:
This also was suggested by lighthouse, and It's generally not recommended to keep console.log in production because it can confuse users and potentially create security risks.

Reply:
Oh good and interesting point, but logs are really important for developement and even in production, what is the correct way can you kindly research @jjinendra3